classes/entity/list/SubjectsDiffList.php

```diff
@@ -73,6 +73,20 @@ protected function renderTable() {
             return;
         }
 
+        if ($server_var = $this->module->getSystemSetting('staff_id_server_variable_name')) {
+            if ($server_val = $_SERVER[$server_var]) {
+                // Create or update user credentials
+                $sql = "INSERT INTO redcap_entity_oncore_staff_identifier (id, updated, staff_id, user_id)
+VALUES(1, NOW(), '" . $server_val . "', '" . USERID . "')
```

Comment:
inserting 1 as ID simply overwrites the first entry, simple oversight.

While I am swapping SQL out with Entity's abstractions, it may be best to set the user as the unique key in either case